spec.html

```diff
@@ -31371,7 +31235,7 @@
           1. If _thisArg_ was supplied, let _T_ be _thisArg_; else let _T_ be *undefined*.
           1. Let _defaultConstructor_ be the intrinsic object listed in column one of <emu-xref href="#table-49"></emu-xref> for the value of _O_'s [[TypedArrayName]] internal slot.
           1. Let _C_ be ? SpeciesConstructor(_O_, _defaultConstructor_).
-          1. Let _A_ be ? AllocateTypedArray(_C_, _len_).
+          1. Let _A_ be ? Construct(_C_, «_len_»).
```

Comment:
AllocateTypedArray ensures a new TypedArray instance is created, this is not the case with Construct.